### DISCUSSION TITLE
CSV upload created too many blank entries

### DISCUSSION THREAD

**Discussion by Ani-MBC** [AntlerVC/firetable#829] [Support / Q&A] [answered]
**Describe the bug**
I uploaded a csv file with less than 200 records. Rowy imported 900+ records, most of them blank.

**Comment by harinij** [comment, marked as answer]
Hi @Ani-MBC, could you share a bit more about this issue you faced? A video recording will help. NOTE: Importing CSV creates a new entry for each row with an auto-generated ID, unless you pick the ID column with which you want to be matched first. In the case you pick an ID column, if a match is found, then it will be used for updating otherwise a new row will be created.
![Screen Shot 2022-09-21 at 10 21 55 am](https://user-images.githubusercontent.com/307298/191387578-1c777184-208a-447a-981d-eeebf2b90aa1.png)

Also, make sure the fields you import and the field types you pick are compatible. One easy way to check this is on the Import modal's preview screen.
